consensus/polybft/consensus_runtime_test.go

```diff
@@ -1659,13 +1659,13 @@ func TestConsensusRuntime_getExitEventRootHash(t *testing.T) {
 		tree, err := NewMerkleTree(encodedEvents)
```

Comment:
Add `t.Parallel()` (both in the scope `t.Run(...)` and `TestConsensusRuntime_getExitEventRootHash` itself).

Also `TestConsensusRuntime_GenerateExitProof` lacks parallel execution (cannot comment on it, because it was already added through prior PR).